xbmc/LangInfo.cpp

```diff
@@ -426,8 +426,15 @@ bool CLangInfo::Load(const std::string& strLanguage)
     return false;
   }
 
+  //Load in the 'Language' information
```

Comment:
all new comments introduced needs a space after slashes as in our code style docs

```suggestion
  // Load in the 'Language' information
```

Reply:
Apologies, Jenkins overlooked this one too.

Reply:
This has been addressed and will be reflected in my next PR update.